src/OrbitGl/Track.cpp

```diff
@@ -110,11 +111,14 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
   float half_label_width = 0.5f * label_width;
   float tab_x0 = x0 + layout_->GetTrackTabOffset();
 
-  Box box(Vec2(tab_x0, y0), Vec2(label_width, -label_height), track_z);
-  ui_batcher->AddBox(box, kBackgroundColor, shared_from_this());
+  const float intent_x0 = tab_x0 + (intent_ * layout_->GetTrackIntentOffset());
+  if (!transparent_) {
```

Comment:
Why is the transparency needed and in particular the case distinction? Or, ask differently: Why is transparency not simply an alpha value in the color?

Reply:
The meaning of is `transparent` is: "Should the track's background (+tab background and rounded corners) be drawn". I could rename it to `draw_background`? Not sure if this is better.
Of course, we could simply adjust the kBackgroundColor in this case, to use an alpha of 0, but I don't feel that this makes things better. WDYT? I am happy to adjust this here.

Reply:
I think `draw_background` is better as it makes the intent clearer. Transparency is closely related to alpha value and is easy to misunderstand. 

Actually, I do like using alpha of 0 even more, as it simplifies the code structure. You should need fewer case distinctions which is usually a win.

Reply:
It should not make a difference here I assume, but I want to point out that alpha blending is disabled when rendering in picking mode.

Reply:
I renamed the member to `draw_background_` and let it simply adjust the the background color's alpha value.